### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ShakedZrihen**
[workerB](https://workerb.linearb.io/v2/badge/collaboration-page?magicLinkId=2x16HfY)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 # Chillbot
 asdftyhbnkl,.;pokjhytrdxzwerfvbhjiol,.
 more docs!
+
```
Comment (ShakedZrihen) [inline README.md:4]: ee
```suggestion

```

File: README.md [hunk 2]
```diff
@@ -1,3 +1,4 @@
...
```
Comment (ShakedZrihen) [inline README.md:4]:
```suggestion
lol
```

File: README.md [hunk 3]
```diff
@@ -1,3 +1,4 @@
 # Chillbot
 asdftyhbnkl,.;pokjhytrdxzwerfvbhjiol,.
 more docs!
+lol
```
Comment (ShakedZrihen) [inline README.md:4]:
```suggestion
lolss
```

**Comment by EladKohavi** [comment]
jkhgfds

File: README.md [hunk 4]
```diff
@@ -1,3 +1,4 @@
 # Chillbot
 asdftyhbnkl,.;pokjhytrdxzwerfvbhjiol,.
```
Comment (EladKohavi) [inline README.md:2]: no

File: README.md [hunk 5]
```diff
@@ -1,3 +1,4 @@
...
 more docs!
+lolss
```
Comment (ShakedZrihen) [inline README.md:4]:
```suggestion
lol
```

**Pull request closed without merging**